docs/developer-docs/getting-started/tokens-and-cycles.mdx

```diff
@@ -0,0 +1,147 @@
+
+## ICP tokens
+
+ICP is the Internet Computer's native utility token. Its value is determined on the open market. ICP tokens play a key role in the governance and economics of the Internet Computer.
+
+There are three primary ways to use ICP tokens:
+
+- Convert ICP into [cycles](#what-are-cycles), which are required to pay for a <GlossaryTooltip>canister's</GlossaryTooltip> resources. **All dapp developers must acquire cycles to deploy their canister onto the mainnet.**
+
+- Trade ICP for other assets, such as other tokens.
+
+- Stake ICP in a [neuron](/docs/current/developer-docs/daos/nns/concepts/neurons/neuron-overview) in return for network governance participation and rewards.
+
+## Cycles
+
+The Internet Computer runs on a network of nodes owned and operated by a community of independent node providers distributed across the globe. Node providers spend money to run and maintain nodes, such that they purchase hardware and pay for the electricity and network bandwidth used by the node. The Internet Computer compensates these node providers on a monthly basis by minting and distributing rewards in the form of ICP tokens.
+
+Canisters consume the resources provided by these nodes when they are deployed on the mainnet. To make ICP sustainable, canisters are required to pay for the resources they use.
+
+Resource consumption is not paid in ICP tokens but **cycles**. As the canister is used, its cycles balance is continuously reduced. Eventually, the canister needs to be "topped up" with more cycles. The canister’s developer is responsible for topping up the canister.
+
+To get cycles, you must convert ICP tokens into cycles.
+
+:::info Why are cycles different from ICP tokens?
+
+While the value of an ICP token is volatile, cycles are not. Cycles are pegged to [XDR](https://en.wikipedia.org/wiki/Special_drawing_rights), a group of fiat currencies. This allows resource consumption and canister execution on ICP to be a rather stable price.
+:::
+
+## Other token types
+
+The Internet Computer supports creating and deploying custom fungible or non-fungible tokens through the [ICRC standards](/docs/current/developer-docs/defi/tokens/token-standards). Each ICRC standard defines a set of parameters that must be met to adhere to the standard. Not all ICRC standards are for tokens; some are for notifications, architectures, or other network functions.
+
+## What is a ledger?
+
+A [**ledger**](/docs/current/developer-docs/defi/tokens/ledger/overview) is a system canister used to store accounts and their transactions. Each developer identity has a **ledger account identifier**.
+
+Each token on the Internet Computer has its **own** ledger. This means that there are separate ledgers for ICP tokens, cycles, and every ICRC token that has been deployed on the network.
+
+[Learn more about ledgers](/docs/current/developer-docs/defi/tokens/ledger/overview).
+
+## Obtaining cycles
```

Comment:
Since we won't be pursuing local ICP as a way to teach about ICP and cycles, I think we should keep this as a core part of the onboarding. Otherwise, they may never return to this section and learn it. 

Also, since we decided that obtaining ICP/cycles is the 'goal' of onboarding, I don't think making it seem like it's an optional step they can skip makes sense.